lib/rubocop/cop/ezcater/feature_flag_active.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Ezcater
+      # Use `EzcaterFeatureFlag.active?` with a tracking_id or array of identifiers
+      #
+      # @example
+      #
+      #   # good
+      #   EzFF.active?("FlagName", tracking_id: "user:12345")
+      #   EzFF.active?("FlagName", identifiers: ["user:12345", "user:23456"])
+      #
+      #   # bad
+      #   EzFF.active?("FlagName")
+
+      class FeatureFlagActive < Cop
+
+        MSG = "`EzFF.active?` must be called with at least one of `tracking_id` or `identifiers`"
+
+        def_node_matcher :ezff_active_one_arg, <<-PATTERN
+          (send
+            (_ _ {:EzFF :EzcaterFeatureFlag}) :active? (str _))
+        PATTERN
+
+        def_node_matcher :args_matcher, <<-PATTERN
```

Comment:
Having never written a cop, this syntax is both confusing but makes sense on further inspection. Neat!

Reply:
Yep! I felt the same way! There's a nifty command line tool `ruby-parse` that was very helpful:

```shell
~$ ruby-parse -e 'EzFF.active?("foo", bar: "baz")'
(send
  (const nil :EzFF) :active?
  (str "foo")
  (kwargs
    (pair
      (sym :bar)
      (str "baz"))))
```